components/03-modules/toolbar/_toolbar.scss

```diff
@@ -1,16 +1,36 @@
-$toolbar__offset               : $spacer !default;
-$toolbar__secondary-text-color : #a09fa6 !default;
-$toolbar__icon-color           : #a09fa6 !default;
-$toolbar__icon-color--active   : $black !default;
-$toolbar__select-border        : 1px solid $black !default;
-$toolbar__select-border-radius : 0 !default;
+$toolbar__dropdown-icon-area-border      : none !default;
```

Comment:
in fact, what is `dropdown-icon` here? And why we keep here `area` and in 2 others we don't have `area`. In other side, i don't know how to change this names

Reply:
as it related to clickable area of icon not the icon itself. is related to select (choices lib)